decorator.rb

```diff
@@ -1,20 +1,6 @@
 require './person'
 
-class Nameable
-    def correct_name
-      raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
-    end
-  end
-
-  class NameComponent < Nameable
-    def correct_name
-      'ConcreteComponent'
-    end
-  end
   class Decorator < Nameable
-    attr_accessor :name
-  
-    # @param [Component] component
     def initialize(nameable)
       @nameable = nameable
     end

```

Comment:
- [ ] Good job so far 👍 Kindly add the `attr_accessor :nameable` to your `Decorator` class 😄